src/language-service/src/schemas/integrations/core/mqtt.ts

```diff
@@ -233,6 +233,12 @@ export interface AlarmControlPanelPlatformSchema extends PlatformSchema {
    */
   command_template?: Template;
 
+  /**
+   * The MQTT topic to publish commands to change the alarm state.
+   * https://www.home-assistant.io/integrations/alarm_control_panel.mqtt/#command_topic
+   */
+   command_topic?: string;
```

Comment:
It is actually required!

https://github.com/home-assistant/core/blob/2cbcd5f2a909d6fb04dc9d03deb6940ba05109f8/homeassistant/components/mqtt/alarm_control_panel.py#L91

```suggestion
   command_topic: string;
```